llvm-hs/Setup.hs

```diff
@@ -170,6 +170,9 @@ main = do
           newHsc buildInfo localBuildInfo =
 #endif
               PreProcessor {
+#if MIN_VERSION_Cabal(3,8,0)
+                  ppOrdering = \_ _ ms -> pure ms,
```

Comment:
Hello! Maybe [`unsorted`](https://hackage.haskell.org/package/Cabal-3.8.1.0/docs/Distribution-Simple-PreProcess.html#v:unsorted) would be better than manual implementation?

Reply:
Yes, you are right. I added a code suggestion.